injector.py

```diff
@@ -1,29 +1,34 @@
 from pywinauto.handleprops import processid
 from pywinauto.handleprops import is64bitprocess
 from pywinauto.timings import TimeoutError
-from pywinauto import Desktop
 from pywinauto import sysinfo
-from socket import *
+from socket import socket
+from socket import AF_INET
+from socket import SOCK_DGRAM
 import win32event
 import cfuncs
 import ctypes
 import sys
 import os
 
+# Relative path, would be changed after adding to pywinauto
+dll_path = "./dll_to_inject/"
+
 class Injector(object):
 
     """Class for injections dll and set hook on windows messages"""
 
-    def __init__(self, process_name, dll_path, is_unicode = False):
-        """Constructor inject dll, set socket and hook (one process - one class instanse"""
-        self.dll_path = os.path.abspath(dll_path)
-        self.app = Desktop(backend="win32")[process_name]
+    def __init__(self, app, is_unicode = False):
+        """Constructor inject dll, set socket and hook (one application - one class instanse)"""
+        self.app = app
         self.is_unicode = is_unicode
         self.pid = processid(self.app.handle)
+        if not sysinfo.is_x64_Python() == is64bitprocess(self.pid):
+            raise RuntimeError("Application and Python must be both 32-bit or both 64-bit")
         self.h_process = self._get_process_handle(self.pid)
-        if (int(sysinfo.is_x64_Python()) + int(is64bitprocess(self.pid))) % 2 != 0:
-            raise RuntimeError("Application and python version must be same versions (x32 or x64)")
-
+        self.dll_path = os.path.abspath("{0}pywinmsg{1}{2}.dll".format(dll_path,
+            "64" if is64bitprocess(self.pid) else "32",
+            "u" if self.is_unicode else "c"))
```

Comment:
I would leave letter `u` for unicode DLLs, but remove letter `c` for ANSI libraries. This is more complaint with standard VC redist libraries names like `msvcr100u.dll` and `msvcr100.dll`